### PR TITLE
Use promises and ES2015ify tests

### PR DESCRIPTION
**Pull request by kevva**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -2,22 +2,18 @@
 var got = require('got');
 var registryUrl = require('registry-url')();
 
-module.exports = function (name, cb) {
+module.exports = function (name) {
 	if (!(typeof name === 'string' && name.length !== 0)) {
 		throw new Error('Package name required');
```
Comment (SamVerschueren) [inline index.js]: Should return a rejected promise instead of throwing the error :).

File: package.json [hunk 2]
```diff
@@ -33,12 +33,18 @@
...
     "registry-url": "^3.0.0"
   },
   "devDependencies": {
-    "mocha": "*"
+    "ava": "*",
+    "xo": "*"
+  },
+  "xo": {
+    "ignore": [
+      "test.js"
```
Comment (SamVerschueren) [inline package.json]: [`ignores`](https://github.com/sindresorhus/xo#ignores) instead of `ignore`.

File: package.json [hunk 3]
```diff
@@ -33,12 +33,18 @@
     "available"
   ],
   "dependencies": {
-    "got": "^3.2.0",
+    "got": "^4.2.0",
     "log-symbols": "^1.0.0",
     "meow": "^3.3.0",
     "registry-url": "^3.0.0"
```
Comment (sindresorhus) [inline package.json]: You need to include `pinkie-promise`.

**Comment by sindresorhus** [comment]
Can you fix the merge conflict? Sorry, I pushed an update forgetting about this PR...

**Comment by kevva** [comment]
Should be resolved now.

**Pull request merged**